nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-processors/src/main/java/org/apache/nifi/processors/iceberg/PutIceberg.java

```diff
@@ -375,4 +364,44 @@ void abort(DataFile[] dataFiles, Table table) {
                 .run(file -> table.io().deleteFile(file.path().toString()));
     }
 
+    public enum UnmatchedColumnBehavior implements DescribedValue {
+        IGNORE_UNMATCHED_COLUMN("IGNORE_UNMATCHED_COLUMN",
```

Comment:
Instead of specifying the `value` as the first argument to the enum, you can use `name()` in the getValue() method.